### ISSUE TITLE
Avatar / icon of buddy does not show up on xmpp client with purple-facebook transport

### ISSUE THREAD

**Issue by jirikeller** [SpectrumIM/spectrum2#235]
Hello,
 this is kind-of duplicate of [https://github.com/dequis/purple-facebook/issues/353](url), but as it seems to be somewhere in between purple-facebook and spectrum2, I am posting now also here. 
  I use purple-facebook as spectrum2 transport for Prosody and contrary to hangout transport my contacts photos do not show up. Is there any easy fix such as configuration option ? I am on Debian Jessie server and use both, purple-facebook and spectrum2, from repository, if this matters. Connecting directly using purple-facebook with pidgin works ok, spectrum2 logs claims that icons are present and up-to-date avatars are downloaded to appropriate /var/lib/spectrum2/fb.mydomain.tld/icons directory. accounts.xml  file is truncated - probably rather result than the source of problems. 

   any help appreciated
     George

**Comment by jirikeller** [comment]
yes, still unsolved. I just tried to move the stuff out of the folder, which did not help, respectively I was not able to "log in", so I put back my original files. Anythink I can test ?

**Comment by dos1** [comment]
Have you tried to start fresh?

**Comment by jirikeller** [comment]
Not sure what you mean by this - unregistering the transport ? I have some contacts "merged" on my phone/carddav server with the transport jabber address, this may cause troubles. But I could register another jabber user and try to connect to the same FB account, if this is what you mean by "start fresh". I tried to move stuff from the fb transport directory, but then it either does not connect (whithout sqlite db) or nothing changed.

**Comment by dos1** [comment]
Create a new transport and test it with new account. If it works, then you know that something just went wrong with your current installation. If it doesn't, you have some issue that appears with your libs/server/client/config combination.
